pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -210,6 +210,16 @@ func (m *kubeGenericRuntimeManager) generateContainerResources(pod *v1.Pod, cont
 	}
 }
 
+// generatePodSandboxResources generates platform specific (linux) podsandox resources config for runtime
+func (m *kubeGenericRuntimeManager) generatePodSandboxResources(sandboxID string, pod *v1.Pod) *runtimeapi.UpdatePodSandboxResourcesRequest {
```

Comment:
the name of this method doesn't really match the return type... This is only called by `updatePodSandboxResources` - maybe just inline it there?